webpack/assets/javascripts/react_app/components/Editor/components/EditorSettings.js

```diff
@@ -74,6 +76,36 @@ const EditorSettings = ({
             </Dropdown.Menu>
           </Dropdown>
         </div>
+        <div className="cog-popover-dropdown">
+          <div className="cog-popover-dropdown-title">
+            {__('Autocompletion')}
+          </div>
+          <div className="dropdown btn-group">
+            <input
+              id="autocompletion-checkbox"
+              name="autocompletion"
+              type="checkbox"
+              checked={autocompletion}
+              onChange={e => changeSetting({ autocompletion: !autocompletion })}
+            />
+          </div>
+        </div>
+        <div className="cog-popover-dropdown">
+          <div className="cog-popover-dropdown-title">
+            {__('Live Autocompletion')}
+          </div>
+          <div className="dropdown btn-group">
+            <input
```

Comment:
Live Auto completion works only if auto completion is enabled, so `#live-autocompletion-checkbox` should be disabled when `autocompletion` is `false`.

Reply:
Thanks, fixed.